src/index.ts

```diff
@@ -30,7 +30,7 @@ export default function prevalPlugin(
         prevalComment.value = ' this file was prevaled'
 
         // @ts-expect-error the types for this is wrong...
-        const {code: string} = transformFromAst(
+        const {code: string}: {code: string} = transformFromAst(
```

Comment:
I'm sneaking this in because I saw the lint warjjng when checking if you needed the Istanbul comments for coverage check.